app/messaging/save-to-database.js

```diff
@@ -1,17 +1,27 @@
 const db = require('../data')
 
+const formatDateToUK = (date) => {
+  const day = String(date.getDate()).padStart(2, '0')
+  const month = String(date.getMonth() + 1).padStart(2, '0')
+  const year = date.getFullYear()
+  return `${day}-${month}-${year}`
```

Comment:
This needs to return a date object as sequelize cant accept a type of string because we defined the type to date.
eg `return new Date(...)`